Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  module_function
+
+  def bump_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `bump` [<options>] [<formula>]
+
+        Display out-of-date brew formulae and the latest version available.
+        Also displays whether a pull request has been opened with the URL.
+      EOS
+      flag "--limit=",
+           description: "Limit number of package results returned."
+      switch :verbose
+      switch :debug
+    end
+  end
+
+  def bump
+    args = bump_args.parse
+
+    requested_formulae = !args.formulae.empty? ? args.formulae.map(&:name) : nil
+
+    requested_limit = args.limit.to_i if args.limit.present?
+
+    repology_data = if requested_formulae
+      response = {}
+      requested_formulae.each do |formula|
+        raise FormulaUnavailableError, formula unless validate_formula(formula)
+
+        package_data = Repology.single_package_query(formula)
+        response[package_data.keys.first] = package_data.values.first if package_data
+      end
+
+      response
+    else
+      Repology.parse_api_response
+    end
+
+    validated_formulae = {}
+
+    validated_formulae = Repology.validate_and_format_packages(repology_data, requested_limit) if repology_data
```

Comment:
Might be nice to output something at this stage otherwise I can see:

```
==> Querying outdated packages from Repology
438 outdated packages found
```

and then an indefinite pause without seeing what is happening.

Reply:
@MikeMcQuaid we previously had some `ohai` output at this stage and it was discussed here:

https://github.com/Homebrew/brew/pull/7834#discussion_r462230052

Ultimately it was removed after this discussion.  I can add it back if you think that's the better choice.

Reply:
Perhaps it's worth adding back only in the cases where it's going to be slow e.g. where you don't have a single formula?